src/storage/store/column_chunk.cpp

```diff
@@ -403,67 +403,59 @@ bool ColumnChunk::sanityCheck() {
 
 ColumnChunkMetadata ColumnChunk::getMetadataToFlush() const {
     KU_ASSERT(numValues <= capacity);
-    StorageValue minValue = {}, maxValue = {};
+    std::optional<StorageValue> minValue, maxValue;
     if (capacity > 0) {
+        std::optional<NullMask> nullMask;
```

Comment:
Are we able to merge the code of computing min/max here with the one (`getMinMax`) inside `Column`? I feel they duplicate the same logic though very minor variations.

Reply:
Yes, that's a good idea.